products/opensuse/main.pm

```diff
@@ -78,10 +78,13 @@ sub cleanup_needles() {
     }
     # unregister christmas needles unless it is December where they should
     # appear. Unused needles should be disregarded by admin delete then
-    my @time = localtime();
-    unregister_needle_tags('CHRISTMAS') unless $time[4] == 11;
+    unregister_needle_tags('CHRISTMAS') unless get_var('WINTER_IS_COMING', '');
 }
 
+# we need some special handling for the openSUSE christmas needles
+my @time = localtime();
+set_var('WINTER_IS_COMING', 1) if $time[4] == 11;
```

Comment:
this calculation is unfortunately off. the actual theme code defines the likeness of the winter theme as:

```
 over month 12 eq { 5 add } if
  over month 1 le { 5 add } if
  over weekday 5 eq { 2 mul } if
  over weekday 6 eq { 4 mul } if
  over dup month 12 eq exch day dup 24 ge exch 6 eq or and { pop 100 } if
  over dup month 1 eq exch day 1 eq and { pop 100 } if
```

So you will have to worry about WINTER in january as well - which is actually not coming at that point, but at its high time :)